src/lib/logger.js

```diff
@@ -177,170 +109,72 @@ class Logger {
     return this;
   }
 
-
-  debug(message, data = null, opts = {}) {
-    this.log(message, data, {
-      level: Severity.DEBUG,
-      ...opts
-    });
-
+  // Options
+  channel(channel) {
+    this.options.channel = channel;
     return this;
   }
 
-  default(message, data = null, opts = {}) {
-    this.log(message, data, {
-      level: Severity.DEFAULT,
-      ...opts
-    });
-
+  enabled(enabled) {
+    this.options.enabled = enabled;
     return this;
   }
 
-  warning(message, data = null, opts = {}) {
-    this.log(message, data, {
-      level: Severity.WARNING,
-      ...opts
-    });
-
+  user(user) {
+    this.options.user = user;
     return this;
   }
 
-  alert(message, data = null, opts = {}) {
-    this.log(message, data, {
-      level: Severity.ALERT,
-      ...opts
-    });
-
-    return this;
-  }
+  level(level) {
+    this.options.level = Severity.registry.get(level);
 
-  emergency(message, data = null, opts = {}) {
-    this.log(message, data, {
-      level: Severity.EMERGENCY,
-      ...opts
-    });
+    this._notifiers.forEach(notif => notif.level(this.options.level));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
    this._notifiers.forEach((notif) => notif.level(this.options.level));
```